pages/jobs/[listing].tsx

```diff
@@ -19,11 +23,36 @@ export const getStaticPaths: GetStaticPaths = async () => {
 };
 
 export const getStaticProps: GetStaticProps<
-  { listing: Listing },
+  { listing: Listing & { contacts: Employee[] } },
   { listing: string }
 > = async (context) => {
   // This will never be empty as that path is caught by 'index.tsx' file
   const fileName = `${context?.params?.listing}.md`;
   const listing = await getListing(fileName);
-  return { props: { listing }, revalidate: 24 * 60 * 60 };
+  const contactEmails = listing.contact_emails?.split(',').map((e) => e.trim());
```

Comment:
Hm. I think this probably should be a part of `getListings.ts`, but OK enough I think .